exercises/poker/src/lib.rs

```diff
@@ -2,6 +2,6 @@
 ///
 /// Note the type signature: this function should return _the same_ reference to
 /// the winning hand(s) as were passed in, not reconstructed strings which happen to be equal.
-pub fn winning_hands<'a>(_: &[&'a str]) -> Option<Vec<&'a str>> {
-    unimplemented!()
+pub fn winning_hands<'a>(input: &[&'a str]) -> Option<Vec<&'a str>> {
```

Comment:
I'd like this to tell me what each element of the slice is, so I'll say `hands` or `input_hands`